spec.md

```diff
@@ -310,6 +310,19 @@ that contains both context and data).
   * MUST be a non-empty string
   * MUST be unique within the scope of the producer
 
+### followsFrom
+* Type: String
+* Description: The eventID of the event that caused this event. The semantics
```

Comment:
one could argue that a set of eventID(s) could be the cause of an event and that this field should allow for this.  allowing delimited eventID(s) to be supplied could satisfy that fact at the cost of complexity